server/admin/http/handler.go

```diff
@@ -394,3 +395,42 @@ func decodePeerID(id string, w http.ResponseWriter) (peer.ID, bool) {
 	}
 	return peerID, true
 }
+
+var healthCheckMH multihash.Multihash
+var healthCheckValue indexer.Value
+
+func init() {
```

Comment:
minor comment, feel free to postpone for future improvements: 
Ideally, the data here should be mutated periodically: if kept fixed, an optimised version of the store could detect no changes and return without performing IO operations. This then reduces our confidence in healthcheck passing.